monitoring/datadog_event.py

```diff
@@ -140,7 +140,7 @@ def post_event(module):
     headers = {"Content-Type": "application/json"}
 
     (response, info) = fetch_url(module, uri, data=json_body, headers=headers)
-    if info['status'] == 202:
+    if (info['status'] == 200) in (info['status'] == 202):
```

Comment:
I don't see the sense in this condition, shouldn't it be more like

~~~python
if info['status'] in [200, 202]:
~~~